login_page/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>login_page</title>
+    <link rel="stylesheet" href="/reset.css"/>
+    <link rel="stylesheet" href="/login_page/style.css" />
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" crossorigin="anonymous">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+    <div class="container">
+        <a class="panda_logo" a href="/index.html">
+            <picture>
+                <source media="(max-width: 768px)" srcset="/login_page/login_image/mobile/panda_logo_mobile.png">
+                <img src="/login_page/login_image/logo.png" alt="판다마켓 로고">
+            </picture>
+        </a>
+        <div class="input_sector">
+            <div class="email_sector">
+                <span>이메일</span>
+                <div class="inputbox">
+                    <input type="email" placeholder="codeit@email.com" >
+                </div>
```

Comment:
❗️ 수정요청
특정 input이 어떤 값을 받는지 설명하는 text들을 label 이라고 합니다.
이러한 친구들은 일반적으로 마우스로 선택시 연결된 input으로 focus가 이동하는 기능이 있습니다~
이를 위해, 그리고 명확한 연결관계 표시하기 위해 label 태그를 사용해주시는 것이 좋습니다~

```suggestion
                <label for="email">이메일</label> // label의 for 속성에 input id 값을 넣어 둘이 연결
                <div class="inputbox">
                    <input type="email" id="email" placeholder="codeit@email.com" >
                </div>
```